pkg/config/system_probe.go

```diff
@@ -76,7 +76,7 @@ func InitSystemProbeConfig(cfg Config) {
 	// network_tracer settings
 	// we cannot use BindEnvAndSetDefault for network_config.enabled because we need to know if it was manually set.
 	cfg.SetKnown(join(netNS, "enabled"))
-	_ = cfg.BindEnv(join(netNS, "enabled"), "DD_SYSTEM_PROBE_NETWORK_ENABLED")
+	cfg.BindEnv(join(netNS, "enabled"), "DD_SYSTEM_PROBE_NETWORK_ENABLED") //nolint:errcheck
```

Comment:
Any reason to switch to the magic comment? I think the explicit error ignore with `_` is more obvious, and doesn't rely upon a linting tool supporting a specific magic comment.